langchain4j-cassandra/src/test/java/dev/langchain4j/store/embedding/astradb/AstraDbEmbeddingStoreIT.java

```diff
@@ -0,0 +1,98 @@
+package dev.langchain4j.store.embedding.astradb;
+
+import com.dtsx.astra.sdk.AstraDB;
+import com.dtsx.astra.sdk.AstraDBClient;
+import dev.langchain4j.data.document.Metadata;
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.store.embedding.EmbeddingMatch;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import java.util.List;
+import java.util.UUID;
+
+import static com.dtsx.astra.sdk.utils.TestUtils.getAstraToken;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@Disabled("AstraDB is not available in the CI")
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@EnabledIfEnvironmentVariable(named = "ASTRA_DB_APPLICATION_TOKEN", matches = "Astra.*")
+class AstraDbEmbeddingStoreIT {
```

Comment:
Please extend all integration tests for both Astra and Cassandra from `EmbeddingStoreIT`, it has a minimal set of tests to ensure that integration is working properly